resources/roca_lib.brs

```diff
@@ -179,7 +179,7 @@ sub __suite_exec(args as object)
     end for
     tap.exitSubTest()
 
-    index = args.index
+    index = 0
```

Comment:
I'm torn about whether this should be `subTestIndex` or not!  If you get time, could you try a structure like this real fast?  I'm not sure if it'd produce a suite and a testcase both with index `1` in the output, and if that's okay per the `tap-parser` module!

```brightscript
describe("foo", sub()
    describe("bar", sub()
        it("sub-test 1", sub()
        end sub)

        it("sub-test 2", sub()
        end sub)
    end sub)

    it("test one", sub()
    end sub)
end sub)
```

Reply:
Sure, gonna give it a try

Reply:
Actually you were right, it didn't work on nested test cases but using `subTestIndex` does the trick